tools/modules/functions/module_default.sh

```
@@ -0,0 +1,106 @@
+}
+
+module_default_enable()
+{
+	local services="$1"
+	echo "Enabling $services..."
+	srv_enable $services
+}
+
+module_default_help()
+{
+	local module=($1) packages="$2" services="$3" extra="$4"
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)